pkg/backup/backup_new_test.go

```diff
@@ -1629,6 +1631,280 @@ func TestBackupWithSnapshots(t *testing.T) {
 	}
 }
 
+// TestBackupWithInvalidHooks runs backups with invalid hook specifications and verifies
+// that an error is returned.
+func TestBackupWithInvalidHooks(t *testing.T) {
+	tests := []struct {
```

Comment:
Is this a table based test just to make it easier to add cases in the future?

Reply:
Yeah, less typing if someone wants to add more cases down the road